appengine/flexible/Datastore/Datastore.Sample/Program.cs

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+using Google.Cloud.Datastore.V1;
+using System.Net;
+
+public class Program {
+	private WebApplication App {get; set;}
+	private DatastoreDb Datastore {get; set;}
+	private KeyFactory VisitKeyFactory {get; set;}
+
+	private Program(string[] args) {
+		var builder = WebApplication.CreateBuilder(args);
+		App = builder.Build();
+		App.MapGet("/", handleGetAsync);
+		Datastore = DatastoreDb.Create(builder.Configuration["GoogleProjectId"]);
+		VisitKeyFactory = Datastore.CreateKeyFactory("visit");
+	}
+
+	private async Task handleGetAsync(HttpContext context) {

```

Comment:
Method names start with capital letters.

```suggestion
	private async Task HandleGetAsync(HttpContext context, DatasotreDb datasore, KeyFactory visitKeyFactory) {

```

Reply:
Done.